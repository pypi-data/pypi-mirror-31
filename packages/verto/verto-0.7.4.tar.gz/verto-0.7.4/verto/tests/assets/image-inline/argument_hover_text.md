An inline image: {image-inline file-path="http://placehold.it/350x150" hover-text="Lipsum Lorem"}.
