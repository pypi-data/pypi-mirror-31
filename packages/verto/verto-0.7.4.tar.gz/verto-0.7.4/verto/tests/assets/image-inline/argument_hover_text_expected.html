<p>An inline image: <div>
<img src="http://placehold.it/350x150" title="Lipsum Lorem" />
</div>.</p>
